docs/source/library-user-guide/using-the-dataframe-api.md

```diff
@@ -19,129 +19,236 @@
 
 # Using the DataFrame API
 
-## What is a DataFrame
+## What is a DataFrame?
 
-`DataFrame` in `DataFrame` is modeled after the Pandas DataFrame interface, and is a thin wrapper over LogicalPlan that adds functionality for building and executing those plans.
-
-```rust
```

Comment:
I moved this example to the end as I think it makes more sense once you see the DataFrame in action